### ISSUE TITLE
non-breaking space is not a right turnstile

### ISSUE THREAD

**Issue by modulovalue** [ianh/owl#41]
(@ianh I hope you don't me reporting any issues I find)

This one is really not urgent, but the following grammar:

```
foo = a 
```

Leads to the following error message:

```
error: 'ￂﾠ' isn't a valid token

  foo = aￂﾠ
         ~~
```

The space after the a is a non-breaking space (command+space), and it is being reported as a right turnstile.

**Comment by ianh** [comment]
Can you clarify what you mean by a right turnstile?  Currently only ASCII white space is interpreted as whitespace when tokenizing.

**Comment by modulovalue** [comment]
<img width="619" alt="Screenshot 2023-11-16 at 17 31 12" src="https://github.com/ianh/owl/assets/24444584/3616a0c0-2e78-44f2-aa57-17303097f02e">

The UI on the rhs says that there's a "ￂ" symbol ([a right turnstile](https://en.wikipedia.org/wiki/Turnstile_(symbol))), but it is a [non-breaking space](https://en.wikipedia.org/wiki/Non-breaking_space)

**Comment by ianh** [comment]
Ah, thanks!  I was missing the context that this was happening in the web tool.

**Comment by ianh** [comment]
Fixed in 75d6bff, thanks.
